### ISSUE TITLE
Create slow:// io plugin

### ISSUE THREAD

**Issue by radare** [radare/radare2#13580]
basically the same as posix, but for debuggnig purposes. it will be printing every read/write that happens and perform a sleep(1) (or usleep), this way we can improve the amount of io accesses. maybe just include this plugin in development cycle

**Comment by condret** [comment]
can't you just just frida for this?

**Comment by trufae** [comment]
Yeah you can do whatever you want with frida.. and actually i think that would make more sense to be an option in io rather than an actual uri wrapper or io plugin. So lets just close it
